.azure-devops/ci-build.yml

```diff
@@ -14,6 +14,8 @@ variables:
     value: 'Arcus.DevOps.Templates'
   - name: 'Arcus_DevOps_Templates_TestValue'
     value: 'Replace token with me!'
+  - name: 'Package.Version'
+    value: '1.2.3'
```

Comment:
This should just be the build version imo

Reply:
The name you mean?

Reply:
No, the value because every PR would try to create the same release but it already exists nor is it easy to troubleshoot